test/test_sympy_utils.py

```diff
@@ -288,6 +288,10 @@ def test_python_interp_fx(self, fn):
         if fn in ("log", "exp"):
             return
 
+        # Sympy does not support truncation on symbolic shapes
+        if fn in ("truncdiv", "mod"):
+            return
+
         vals = CONSTANTS
```

Comment:
NIT: I notice `-1` is the only negative constant. It would be good if there were some larger ones so the modulo isn't just 0.